app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -1108,9 +1752,10 @@ class ExplorationActivityTest {
       WorkManagerConfigurationModule::class, HintsAndSolutionConfigModule::class,
       FirebaseLogUploaderModule::class, FakeOppiaClockModule::class, PracticeTabModule::class,
       DeveloperOptionsStarterModule::class, DeveloperOptionsModule::class,
-      ExplorationStorageModule::class
+      TestExplorationStorageModule::class
```

Comment:
Do we need to update this in all test files? if any?

Reply:
` TestExplorationStorageModule` is added to the test file where we need to change the size of the checkpoint database to test a few specific checkpointing features. 

The test files that need this module are:

1. ExplorationActivityTest
2. ExplorationDataControllerTest
3. ExplorationProgressControllerTest
4. ExplorationCheckpointControllerTest

Reply:
Are all test in this class passing on robolectric?

Reply:
Yes, all tests added here are passing on robolectric